### ISSUE TITLE
Is there an Assembly Manual for the ERP?

### ISSUE THREAD

**Issue by MADISON948** [EtteGit/EnragedRabbitProject#196]
Hello, I would like to know if there is an ERP assembly manual.

**Comment by StanislavJochman** [comment]
@MADISON948 
Yes, there is in [folder documentation](https://github.com/EtteGit/EnragedRabbitProject/blob/main/Documentation/ERCF_Manual.pdf~)
